src/Sylius/Component/Variation/Model/Option.php

```diff
@@ -24,52 +24,35 @@
 class Option extends AbstractTranslatable implements OptionInterface
 {
     /**
-     * Property id.
-     *
      * @var mixed
      */
     protected $id;
 
     /**
-     * Internal name.
-     * See description in interface.
-     *
      * @var string
      */
     protected $name;
 
     /**
-     * Presentation.
-     * Displayed to user.
```

Comment:
The line above should be removed, but this line should stay (distinguishes from `name`).